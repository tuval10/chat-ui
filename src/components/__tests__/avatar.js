import React from 'react';
import {create} from 'react-test-renderer';
import Avatar from '../avatar';
import {user} from '../../test_helpers/fixtures'
const {avatar} = user;

describe('Avatar component', () => {
  it('renders correctly', () => {
    const tree = create(<Avatar avatar={avatar} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});