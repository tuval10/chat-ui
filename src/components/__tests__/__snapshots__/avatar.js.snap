// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar component renders correctly 1`] = `
<div
  className="avatar-container"
>
  <img
    alt="user-logo"
    className="avatar-icon"
    src="https://spotim-demo-chat-server.herokuapp.com/avatars/005-bullbasaur.png"
  />
</div>
`;
